src/sentry/api/serializers/models/apitoken.py

```diff
@@ -27,6 +27,13 @@ def serialize(self, obj, attrs, user):
             "state": attrs.get("state"),
         }
         if not attrs["application"]:
-            data["token"] = obj.token
+            include_token = kwargs.get("include_token", True)
```

Comment:
This should default to false in the future, but currently is true because other endpoints and services are also utilizing this serializer

Reply:
rather than passing `include_token` in the kwargs, can we be explicit in the args, and just default it to False if that's what we expect?

Reply:
This is related to the comment above, but in relation to the `False` matter:
Unfortunately we cannot default to `False`, as other API endpoints are referencing this serializer for their contract, and we would need to change more domain code than necessary for this change. This is the result of not using repository patterns and proper contracts/DTOs :/

Reply:
Here's another service/endpoint that uses the same serliazier: https://github.com/getsentry/sentry/blob/cd404411c7b52154f5aca73af93e74b908b3ff27/src/sentry/api/endpoints/integrations/sentry_apps/internal_app_token/index.py#L69